app/component/itinerary.scss

```diff
@@ -466,3 +470,31 @@ div.itinerary-container-content {
 .desktop .itinerary-intermediate-stop-name {
   font-size: 15px;
 }
+
+.itinerary-tab .itinerary-timeframe {
+  padding-top: 15px;
+}
+
+.timeframe-nextday {
+  background-color: $gray;
+  line-height: 1;
+  @include font-narrow-medium;
+  color: $white;
+  border-radius: 2px;
+  padding: 4px 5.5px;
```

Comment:
Since moving to a layout that scales within the breakpoint, relative units are generally suggested. Here the challenge is to select one of the padding variables that closely corresponds with the spec at the intended window size (iphone 5). If a new padding variable is unavoidable, feel free to add it.